File: go-client-sdk/pkg/models/shared/projectrunresponsepayload.go
package shared

// ProjectRunResponsePayload
// Response format returned by the runProject endpoint
type ProjectRunResponsePayload struct {
	ProjectID      string `json:"projectId"`
	ProjectVersion int    `json:"projectVersion"`
	RunID          string `json:"runId"`
	RunStatusURL   string `json:"runStatusUrl"`
	RunURL         string `json:"runUrl"`
	TraceID        string `json:"traceId"`
}

File: typescript-client-sdk/src/sdk/models/shared/projectrunresponsepayload.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { Expose } from "class-transformer";


// ProjectRunResponsePayload
/** 
 * Response format returned by the runProject endpoint
**/
export class ProjectRunResponsePayload extends SpeakeasyBase {
  @SpeakeasyMetadata()
  @Expose({ name: "projectId" })
  projectId: string;

  @SpeakeasyMetadata()
  @Expose({ name: "projectVersion" })
  projectVersion: number;

  @SpeakeasyMetadata()
  @Expose({ name: "runId" })
  runId: string;

  @SpeakeasyMetadata()
  @Expose({ name: "runStatusUrl" })
  runStatusUrl: string;

  @SpeakeasyMetadata()
  @Expose({ name: "runUrl" })
  runUrl: string;

  @SpeakeasyMetadata()
  @Expose({ name: "traceId" })
  traceId: string;
}